src/components/VgtFilterRow.vue

```diff
@@ -134,6 +137,10 @@ export default {
       return placeholder;
     },
 
+    getName(column) {
+      return `vgt-' + ${column.field}`;
```

Comment:
Im not the project owner. But I noticed this. Your mixing template strings with + '

Reply:
Thanks for noticing, I've fixed it.